src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,11 @@
+$date-box-side: 100px;
+$calendar-width: (7 * $date-box-side + 6px);
+$week-days: (
+  "mon" ($date-box-side * 0),
+  "tue" ($date-box-side + 1px),
+  "wed" ($date-box-side * 2 + 2px),
+  "thu" ($date-box-side * 3 + 3px),
+  "fri" ($date-box-side * 4 + 4px),
+  "sat" ($date-box-side * 5 + 5px),
+  "sun" ($date-box-side * 6 + 6px)
```

Comment:
Will be better to make these calculations in the loop, as you already have these values (0-6) there
In case if you need to change the logic, it will be easier to change it in one place in the loop, instead of refactoring the values for all this variables

Reply:
Thanks for your edits, I will definitely take them into account in future projects.